api/types/accesslist/accesslist.go

```diff
@@ -157,11 +253,53 @@ func (a *AccessList) CheckAndSetDefaults() error {
 		return trace.BadParameter("owners are missing")
 	}
 
-	if a.Spec.Audit.Frequency == 0 {
-		return trace.BadParameter("audit frequency must be greater than 0")
+	// Frequency should be converted into a recurrence.
+	if a.Spec.Audit.Frequency != 0 {
+		if a.Spec.Audit.Recurrence.Frequency == 0 && a.Spec.Audit.Recurrence.DayOfMonth == 0 {
+			// Just assume the first of the month.
+			a.Spec.Audit.Recurrence.DayOfMonth = FirstDayOfMonth
+
+			numMonths := int(a.Spec.Audit.Frequency / averageNumberOfHoursInAMonth)
```

Comment:
Not sure I understand, if we're doing specific presets, why do we need to calculate anything?

Reply:
This is trying to take the existing frequency field and converting it to a preset. Basically trying to be backwards compatible here.

Reply:
Do we need to be backwards compatible? Can we deprecate the old field?